airbyte-server/src/main/java/io/airbyte/server/CorsFilter.java

```diff
@@ -1,29 +0,0 @@
-/*
- * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
- */
-
-package io.airbyte.server;
-
-import com.google.common.collect.ImmutableMap;
-import com.google.common.net.HttpHeaders;
-import java.util.Map;
-import javax.ws.rs.container.ContainerRequestContext;
-import javax.ws.rs.container.ContainerResponseContext;
-import javax.ws.rs.container.ContainerResponseFilter;
-
-// https://medium.com/@Leejjon_net/how-to-allow-cross-origin-requests-in-a-jax-rs-microservice-d2a6aa2df484
-public class CorsFilter implements ContainerResponseFilter {
-
-  public static final ImmutableMap<String, String> MAP = ImmutableMap.of(
```

Comment:
for my knowledge - how do we turn this on for Micronaut?

Reply:
https://docs.micronaut.io/latest/guide/#cors